src/shared/components/PostDialog.jsx

```
@@ -1,5 +1,140 @@
 //TODO: 게시글 작성 모달 컴포넌트 구현
+import {
+  Dialog,
+  DialogTrigger,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogDescription,
+  DialogFooter,
+  DialogClose,
```

ESLint

[error] 'DialogClose' is defined but never used.

(no-unused-vars)